logic/Server/ArgumentOption.cs

```diff
@@ -30,12 +30,18 @@ public class ArgumentOptions
         [Option("maxStudentCount", Required = false, HelpText = "The max number of students, 4 by default")]
         public ushort MaxStudentCount { get; set; } = 4;
 
+        [Option("maxTrickerCount", Required = false, HelpText = "The max number of trickers, 1 by default")]
+        public ushort MaxTrickerCount { get; set; } = 1;
+
         [Option('g', "gameTimeInSecond", Required = false, HelpText = "The time of the game in second, 10 minutes by default")]
         public uint GameTimeInSecond { get; set; } = 10 * 60;
 
         [Option('f', "fileName", Required = false, HelpText = "The file to store playback file or to read file.")]
         public string FileName { get; set; } = DefaultArgumentOptions.FileName;
 
+        [Option('s', "spectator", Required = false, HelpText = "Whether allow a spectator to watch the game.")]
+        public bool spectator { get; set; } = true;
```

Comment:
```suggestion
        public bool spectator { get; set; } = true;
        public bool AllowSpectator { get; set; } = true;
```
属性用大驼峰。建议改为 `AllowSpectator`